src/EventStore.Client.Tests/Streams/subscribe_to_all_filtered_live.cs

```diff
@@ -84,8 +91,10 @@ public class subscribe_to_all_from_filtered : IClassFixture<subscribe_to_all_fro
 			var result = new List<ResolvedEvent>();
 			var source = new TaskCompletionSource<bool>();
 
-			_fixture.Client.SubscribeToAll(EventAppeared,
-				false, filter: new EventTypeFilter(new RegularFilterExpression(new Regex($"^{eventTypePrefix}"))));
+			_fixture.Client.SubscribeToAll(Position.End, EventAppeared, false,
+				filter: new EventTypeFilter(new RegularFilterExpression(new Regex($"^{eventTypePrefix}"))));
+
+			await Task.Delay(100);
```

Comment:
Consider raising an issue so that the client gets notified when the subscription has actually been setup server side

Reply:
We can; this would require a change to the proto.